src/main/common/maths.c

```diff
@@ -111,7 +111,7 @@ int32_t applyDeadband(int32_t value, int32_t deadband)
     return value;
 }
 
-inline int constrain(int amt, int low, int high)
```

Comment:
The source for constrain must be in a .h file for other .c files to use it.  To inline, compilers must have the source for the function available.  Inline is only a suggestion, so it's not actually being inlined, except for maybe functions inside of maths.c.

If you want it to be inlined, we can move the code to maths.h.